cfme/automate/buttons.py

```diff
@@ -860,11 +873,23 @@ def create(
                 "assign_buttons": assign_buttons,
             }
         )
+        if not changed:
+            # the form wasn't filled with anything
+            # no button group to instantiate, nothing was created
+            return None
+        try:
+            # add button slow to enable?
+            wait_for(lambda: not view.add_button.disabled,
+                     timeout=5,
+                     handle_exception=True)
+        except TimedOutError:
+            logger.exception('Timed out waiting for add button on button group form')
+            raise CFMEException('Custom button group add form button did not activate')
         view.add_button.click()
         view = self.create_view(ButtonGroupObjectTypeView)
 
         view.flash.assert_no_error()
-        view.flash.assert_message('Button Group "{}" was added'.format(hover))
+        view.flash.assert_success_message('Button Group "{}" was added'.format(hover))
```

Comment:
This should be in the crud test but out of scope for this PR

Reply:
Agreed, I'd like @digitronik to address this, as well as the 5.10.12 failures in PRT for `ButtonCollection.create()`